pydantic_ai_slim/pydantic_ai/models/cohere.py

```diff
@@ -223,6 +225,14 @@ def _map_messages(self, messages: list[ModelMessage]) -> list[ChatMessageV2]:
                         pass
                     elif isinstance(item, ToolCallPart):
                         tool_calls.append(self._map_tool_call(item))
+                    elif isinstance(item, ServerToolCallPart):  # pragma: no cover
+                        # ServerToolCallPart represents a tool call from a remote server
+                        # This is currently never returned from cohere
+                        pass
+                    elif isinstance(item, BuiltinToolReturnPart):  # pragma: no cover
+                        # BuiltinToolReturnPart represents a tool return from a remote server
```

Comment:
Like I mentioned elsewhere, let's drop this line as the "remote server" bit is potentially confusing and the class name is clear enough